requirements-devel.txt

```diff
@@ -4,6 +4,8 @@ coverage
 pytest
 sphinx
 sphinx_rtd_theme
+pytest
+pytest-cov
```

Comment:
what for if `pytest` is already right there above (for no good reason) and both are installed anyways as part of `.[devel]`?

Reply:
Duplicated `pytest` should indeed be removed. I put the dependencies in `requirements-devel.txt` because that was used in some configurations to install dependencies. If that is no longer the case, we should definitely prevent duplication and remove it here.